proto/hub.proto

```diff
@@ -1,34 +1,39 @@
 syntax = "proto3";
 
-import "insonmnia.proto";
+import "proto/insonmnia.proto";
 
 package sonm;
 
 service Hub {
-    rpc Ping(PingRequest) returns (PingReply) {}
-    rpc Status(HubStatusRequest) returns (HubStatusReply) {}
-    rpc List(ListRequest) returns (ListReply) {}
-    rpc Info(HubInfoRequest) returns (InfoReply) {}
+    // Ping checks Hub availability for user
+    rpc Ping (PingRequest) returns (PingReply) {}
+    // Status return Hub status
+    rpc Status (HubStatusRequest) returns (HubStatusReply) {}
 
-    rpc StartTask(HubStartTaskRequest) returns (HubStartTaskReply) {}
-    rpc StopTask(StopTaskRequest) returns (StopTaskReply) {}
+    // MinerList returns Miners connected to Hub
+    rpc MinerList (MinerListRequest) returns (MinerListReply) {}
+    // MinerInfo returns detailed info about conected Miner
+    rpc MinerInfo (MinerInfoRequest) returns (MinerStatusReply) {}
 
-    rpc TaskStatus(TaskStatusRequest) returns (TaskStatusReply) {}
-    rpc MinerStatus(HubStatusMapRequest) returns (StatusMapReply) {}
-    rpc TaskLogs(TaskLogsRequest) returns (stream TaskLogsChunk) {}
+    // TaskXXX forward task operations to given Miner
+    rpc TaskStart (HubStartTaskRequest) returns (HubStartTaskReply) {}
+    rpc TaskStop (TaskStopRequest) returns (TaskStopReply) {}
+    rpc TaskStatus (TaskDetailsRequest) returns (TaskDetailsReply) {}
+    rpc TaskList (HubTaskListMapRequest) returns (TaskDetailsMapReply) {}
+    rpc TaskLogs (TaskLogsRequest) returns (stream TaskLogsChunk) {}
 }
 
-message ListRequest {
+message MinerListRequest {
 }
 
-message ListReply {
+message MinerListReply {
     message ListValue {
         repeated string values = 1;
     }
     map<string, ListValue> info = 1;
 }
 
-message HubInfoRequest {
+message MinerInfoRequest {
```

Comment:
It's a request to Hub, that's why it it prefixed with "Hub". It's difficult to avoid ambiguity in this case in either case, so again why do we need renaming